aten/src/THC/THCTensor.cpp

```diff
@@ -185,12 +185,20 @@ void THCTensor_squeeze1d(THCState *state, THCTensor *self, THCTensor *src, int d
 
   if(src->size(dimension) == 1)
   {
+    std::vector<int64_t> newSize(self->dim() - 1);
```

Comment:
Previously, this function didn't do any dynamic allocations.

Now, there's always a new dynamic allocation. It would be nice to do one of the following:
1) audit the callsites of this function (I don't think there are a lot) and check that those functions aren't important
2) benchmark this change
3) use `SmallVector<int64_t,5>` to avoid performance regressions

Reply:
OK, let's do a SmallVector :)